addons/test_website/tests/test_reset_views.py

```diff
@@ -104,8 +111,9 @@ def test_08_reset_specific_page_view_hard_mode(self):
         # Break it again to have a previous arch different than file arch
         break_view(self.test_page_view.with_context(website_id=1))
         self.assertEqual(total_views + 1, self.View.search_count([('type', '=', 'qweb')]), "Missing COW view")
-        with self.assertRaises(AssertionError):
-            # soft reset should not be able to reset the view as previous
-            # version is also broken
-            self.fix_it('/test_page_view')
+
+        # soft reset should not be able to reset the view as previous
+        # version is also broken
```

Comment:
Don't hesitate to adapt comment when modifying indentation (to keep using all space up to 80)